schema/post.graphql

```diff
@@ -0,0 +1,42 @@
+type Post {
+    id: ID!
+    authorID: ID!
+    content: String!
+    createdAt: Int
+    updatedAt: Int
+    deletedAt: Int
+}
+
+input PostFilter {
```

Comment:
This is not used anywhere?

Reply:
No not at this moment, same as `AuthorFilter`